src/renderer/src/components/song/song.utils.ts

```diff
@@ -295,6 +303,7 @@ window.api.listen("queue::songChanged", async (s) => {
   setSong(s);
   await window.api.request("discord::play", s);
   await play();
+  player.playbackRate = speedNum;
```

Comment:
Why do we have two variables to track the speed, can't we just use the "speed" signal here?

Reply:
Thanks for catching that, I accidentally kept that in there while debugging something else. This should be fixed in https://github.com/Team-BTMC/osu-radio/pull/103/commits/d75d755d27420d35a91a567ceb08539efc5085b9